bot/resources/evergreen/trivia_quiz.json

```diff
@@ -604,5 +604,309 @@
+      "answer": "D"
+    },
+    {
+      "id": 421,
+      "question": "What is a HTTP/1.1 feature that was superseded by HTTP/2 multiplexing and is unsupported in most browsers nowadays?",
+      "answer": "pipelining"
+    },
+    {
+      "id": 422,
+      "question": "Which of these languages is the oldest? Tcl, Smalltalk 80, haskell, standard ML, Java",
+      "answer": "Smalltalk 80"
+    },
+    {
+      "id": 423,
+      "question": "What is the name for unicode codepoints that do not fit into 16 bits",
+      "answer": "surrogates"
+    },
+    {
+      "id": 424,
+      "question": "Under what locale does making a string lowercase behave differently",
+      "answer": "Turkish"
+    },
+    {
+      "id": 425,
+      "question": "What does the \"a\" represent in a HSLA color value?",
+      "answer": "transparency, translucency, alpha value, alpha channel"
+    },
+    {
+      "id": 426,
+      "question": "What is the section of a GIF that is limited to 256 colors called?",
+      "answer": "image block"
+    },
+    {
+      "id": 427,
+      "question": "What is an interpreter capable of interpreting itself called?",
+      "answer": "metainterpreter"
+    },
+    {
+      "id": 428,
+      "question": "Due to what data storage medium did old programming languages, such as cobol, ignore all characters past the 72nd column?",
+      "answer": "punch cards"
+    },
+    {
+      "id": 429,
+      "question": "Which of these sorting algorithms is not stable? Counting sort, quick sort, insertion sort, tim sort, bubble sort",
+      "answer": "quick, quick sort"
+    },
+    {
+      "id": 430,
+      "question": "Which of these languages is the youngest? Lisp, Python, Java, Haskell, Prolog, Ruby, Perl",
+      "answer": "Java"
+    }
+  ],
+  "python": [
+    {
+      "id": 501,
+      "question": "Is it all objects (y/n)?",
+      "answer": "y, yes"
+    },
+    {
+      "id": 502,
+      "question": "Name the only non-dunder method of the builtin slice object.",
+      "answer": "indices"
+    },
+    {
+      "id": 503,
+      "question": "What exception, other than `StopIteration`, can you raise from a `__getitem__` dunder to indicate to an iterator that it should stop?",
+      "answer": "IndexError"
+    },
+    {
+      "id": 504,
+      "question": "What type does the `&` operator return given 2 `dict_keys` objects?",
+      "answer": "set"
+    },
+    {
+      "id": 505,
+      "question": "Can you pickle a running `list_iterator` (y/n)?",
+      "answer": "y, yes"
+    },
+    {
+      "id": 506,
+      "question": "What attribute of a closure contains the value closed over?",
+      "answer": "cell_contents"
+    },
+    {
+      "id": 507,
+      "question": "What name does a lambda function have?",
+      "answer": "<lambda>"
+    },
+    {
+      "id": 508,
+      "question": "What folder contains all special site builtins, such as help or credits?",
```

Comment:
Isn't [`_sitebuiltins`](https://github.com/python/cpython/blob/main/Lib/_sitebuiltins.py) as file, not a folder?

Reply:
will change, resolved :D